Sources/Sentry/SentryCrashReportSink.m

```diff
@@ -33,7 +35,15 @@ - (void)handleConvertedEvent:(SentryEvent *)event
                  sentReports:(NSMutableArray *)sentReports
 {
     [sentReports addObject:report];
-    [SentrySDK captureCrashEvent:event];
+    SentryScope *scope = [[SentryScope alloc] initWithScope:SentrySDK.currentHub.scope];
+
+    if (report[SENTRYCRASH_REPORT_SCREENSHOT_ITEM]) {
+        for (NSString *ssPath in report[SENTRYCRASH_REPORT_SCREENSHOT_ITEM]) {
+            [scope addAttachment:[[SentryAttachment alloc] initWithPath:ssPath]];
```

Comment:
`h`: When do we delete the screenshots? The attachments will be read when the SDK creates the envelope. How do we know that the screenshots are still stored on disk? Maybe it's better to read the attachments already here, and use `SentryAttachment .initWithData`. Then we could delete them already here.